src/gpu/nvidia/cudnn_convolution_impl.hpp

```diff
@@ -581,8 +622,32 @@ struct cudnn_convolution_impl_fwd_t : public cudnn_convolution_impl_base_t {
             CUDA_EXECUTE_FUNC(cuMemcpy, (CUdeviceptr)&host_dst_scale,
                     (CUdeviceptr)dst_scale, sizeof(float));
             float inv_scale = 1.0f / host_dst_scale;
-            CUDNN_EXECUTE_FUNC(
-                    cudnnScaleTensor, handle, descs[io::y], y, &inv_scale);
+            if (data_types[io::y] == CUDNN_DATA_INT8) {
+                cudnnOpTensorDescriptor_t opTensorDesc;
+                CUDNN_EXECUTE_FUNC_V(
+                        cudnnCreateOpTensorDescriptor, &opTensorDesc);
+
+                cudnnOpTensorOp_t opTensorOp = CUDNN_OP_TENSOR_ADD;
+                cudnnDataType_t opTensorCompType = CUDNN_DATA_FLOAT;
+                cudnnNanPropagation_t opTensorNanOpt = CUDNN_NOT_PROPAGATE_NAN;
+                CUDNN_EXECUTE_FUNC_S(cudnnSetOpTensorDescriptor, opTensorDesc,
+                        opTensorOp, opTensorCompType, opTensorNanOpt);
+
+                float alpha_beta = 0.0f;
+                CUDNN_EXECUTE_FUNC(cudnnOpTensor, handle, opTensorDesc,
+                        &inv_scale, y_fp32_desc, y_fp32_data, &alpha_beta,
+                        y_fp32_desc, y_fp32_data, &alpha_beta, descs[io::y], y);
+
+                if (y_fp32_desc)
+                    CUDNN_EXECUTE_FUNC_V(
+                            cudnnDestroyTensorDescriptor, y_fp32_desc);
```

Comment:
This object is created once at primitive creation stage (in `init_scratchpad`). Deallocation should happen in primitive implementation destructor to keep the object alive through primitive object lifetime. With the deallocation in the current  location if user calls execution more than once it will fail, because at the first execution the object was destroyed.